TYPO3.Neos/Configuration/Routes.yaml

```diff
@@ -3,6 +3,17 @@
 #                                                                        #
 
 ##
+# MediaBrowser
+-
+  name: 'Neos Media Browser'
+  uriPattern: 'neos/module/media/browser/<NeosMediaBrowserSubroutes>'
+  subRoutes:
+    'NeosMediaBrowserSubroutes':
+      package: 'Neos.Media.Browser'
+      variables:
+        'defaultUriSuffix': '.html'
```

Comment:
instead we should use the routes by settings feature and put these in the media browser package